app-modules/student-data-model/src/Filament/Resources/StudentResource/Pages/Concerns/HasStudentHeader.php

```diff
@@ -38,17 +38,18 @@
 
 use App\Settings\DisplaySettings;
 use Illuminate\Contracts\View\View;
-use AdvisingApp\Notification\Filament\Actions\SubscribeHeaderAction;
-use AdvisingApp\StudentDataModel\Filament\Resources\StudentResource;
+use AdvisingApp\Prospect\Filament\Resources\ProspectResource;
 use AdvisingApp\StudentDataModel\Settings\StudentInformationSystemSettings;
+use AdvisingApp\Prospect\Filament\Resources\ProspectResource\Pages\ViewProspect;
 use AdvisingApp\StudentDataModel\Filament\Resources\StudentResource\Pages\ViewStudent;
-use AdvisingApp\StudentDataModel\Filament\Resources\StudentResource\Actions\StudentTagsAction;
-use AdvisingApp\StudentDataModel\Filament\Resources\StudentResource\Actions\SyncStudentSisAction;
 
 trait HasStudentHeader
 {
     public function getHeader(): ?View
     {
+        $resource = static::getResource();
+        $name = $resource === ProspectResource::class ? 'prospect' : 'student';
```

Comment:
Let's change this to a match statement for Prospect and Student resource.

Reply:
I think originally the intention was to copy this trait as HasProspectHeader, can you do that @amit-canyon ?